scripts/prep.sh

```diff
@@ -58,33 +55,110 @@ printf "${PURPLE}Starting FireFly Stack: $NEW_STACK_NAME...\n${NC}"
+  heartbeatInterval: 5s
+
+instances:
+  - name: ff0-broadcast
+    test: msg_broadcast
+    length: 5m
+    sender: ${FIRST_NODE_IDENTITY}
+    workers: 10
+    messageOptions:
+      longMessage: true
+  - name: ff0-ff1-msg-private
+    test: msg_private
+    length: 5m
+    sender: ${FIRST_NODE_IDENTITY}
+    recipient: ${SECOND_ORG_IDENTITY}
+    workers: 10
+    messageOptions:
+      longMessage: true
+  - name: ff1-blob-broadcast
+    test: blob_broadcast
+    length: 5m
+    sender: ${SECOND_NODE_IDENTITY}
+    workers: 10
+    messageOptions:
+      longMessage: true
+  - name: ff1-ff0-blob-private
+    test: blob_private
+    length: 5m
+    sender: ${SECOND_NODE_IDENTITY}
+    recipient: ${FIRST_ORG_IDENTITY}
+    workers: 10
+    messageOptions:
+      longMessage: true
+EOF
+
 if [ "$BLOCKCHAIN_PROVIDER" == "geth" ]; then
     output=$(ff deploy ethereum $NEW_STACK_NAME ./firefly/test/data/simplestorage/simple_storage.json | jq -r '.address')
     prefix='contract address: '
     CONTRACT_ADDRESS=${output#"$prefix"}
     FLAGS="$FLAGS -a $CONTRACT_ADDRESS"
     JOBS="$JOBS token_mint custom_ethereum_contract"
+    cat <<EOF >> $BASE_PATH/instances.yml
+  - name: ff0-ff1-mint
+    test: token_mint
+    length: 5m
+    sender: ${FIRST_NODE_IDENTITY}
+    recipient: ${SECOND_ORG_IDENTITY}
+    recipientAddress: ${SECOND_ORG_ADDRESS}
+    workers: 10
+    tokenOptions:
+      tokenType: fungible
+  - name: ff1-contract
+    test: custom_ethereum_contract
+    length: 5m
+    sender: ${SECOND_NODE_IDENTITY}
+    workers: 10
+    contractOptions:
+      address: ${CONTRACT_ADDRESS}
+EOF
 fi
 
 if [ "$BLOCKCHAIN_PROVIDER" == "fabric" ]; then
     docker run --rm -v $BASE_PATH/firefly/test/data/assetcreator:/chaincode-go hyperledger/fabric-tools:2.4 peer lifecycle chaincode package /chaincode-go/package.tar.gz --path /chaincode-go --lang golang --label assetcreator
-    output=$(ff deploy fabric $NEW_STACK_NAME ./firefly/test/data/assetcreator/package.tar.gz firefly assetcreator 1.0)
-    FLAGS="$FLAGS --channel firefly --chaincode assetcreator"
-    JOBS="$JOBS custom_fabric_contract"
+    output=$(ff deploy $NEW_STACK_NAME ./firefly/test/data/assetcreator/package.tar.gz firefly assetcreator 1.0)
+    cat <<EOF >> $BASE_PATH/instances.yml
+  - name: ff1-contract
+    test: custom_fabric_contract
+    length: 5m
+    sender: ${SECOND_NODE_IDENTITY}
+    workers: 10
+    contractOptions:
+      channel: firefly
+      chaincode: assetcreator
+EOF
 fi
 
 echo "FLAGS=$FLAGS"
 
-printf "${PURPLE}Modify the command below and run...\n${NC}"
+printf "${PURPLE}Modify $BASE_PATH/instances.yml and the commnd below and run...\n${NC}"
 
 echo '```'
-printf "${GREEN}nohup ffperf run-tests $JOBS -l 500h -r \"$ORG_IDENTITY\" -x \"$ORG_ADDRESS\" -w 200 -s ~/.firefly/stacks/$NEW_STACK_NAME/stack.json $FLAGS &> ffperf.log &${NC}\n"
+printf "${GREEN}nohup ffperf run -c $BASE_PATH/instances.yml -n ff0-broadcast &> ffperf.log &${NC}\n"
```

Comment:
Worth noting, the `instances.yml` file enables configuring a different number of workers per test type now _however_ it will require running a separate `ffperf` process for each test so be sure to make separate log files.